BSIS-378 Rename general config

bausmeier opened the pull request:
Replaces references to "confirmatory" tests with "repeat" tests.

timler commented:
This code looks good to me and I've tested on my laptop - all fine. 

I'm wondering if best to leave the static const 'DEFER_DONORS_WITH_NEG_CONFIRMATORY_OUTCOMES' unrenamed, I think it is, but I'm not sure.

Happy to press the Merge button, please let me know.

bausmeier commented:
Oh I missed that one even though I changed the value. I think it should be
renamed as well. I will make that change.
On 02 Feb 2016 21:48, "Dagmar Timler" <notifications@github.com> wrote:

> This code looks good to me and I've tested on my laptop - all fine.
>
> I'm wondering if best to leave the static const
> 'DEFER_DONORS_WITH_NEG_CONFIRMATORY_OUTCOMES' unrenamed, I think it is, but
> I'm not sure.
>
> Happy to press the Merge button, please let me know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jembi/bsis/pull/472#issuecomment-178783904>.
>

bausmeier commented:
@timler Renamed that constant. Feel free to merge if you are happy with it.
